src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -171,6 +171,23 @@ public void Should_blow_up_if_name_is_null()
             Assert.Throws<ArgumentNullException>(() => creator.CreateQueueIfNecessary(bindings, WindowsIdentity.GetCurrent().Name));
         }
 
+        [Test]
+        public void Verify_msmq_permissions_and_delete_has_lower_lenght_limit_compared_to_queue_name()
+        {
+            //increase to 50 and create will work but SetPermission and Delete will blow up
```

Comment:
@ramonsmits should we raise an issue to document that older version swallow setpermission issues and that 6.1 will fix that?